README.md

```diff
@@ -253,6 +265,8 @@ The Symbolication Server includes its own tests. There are two options for
 running tests. `python runTests.py` runs all tests.
 `python tests/test_[name].py` runs just one test.
 
+If you have docker, you can run all the tests using `make test`.
+
```

Comment:
There's also a `make lint` rule which will lint the code. I didn't know whether that should get documented or not.